README.md

```diff
@@ -31,10 +31,10 @@ the node package manager `npm`. Then, from inside the `client-js` directory you
 global and project-level dependencies:
 
 ```
-$ npm install -g gulp typescript jspm bower
+$ npm install gulp typescript jspm bower
```

Comment:
Looks like we need to add `bower` to `devDependencies` in `package.json`, but gulp and typescript are already there.